dbt/adapters/trino/connections.py

```diff
@@ -64,7 +62,7 @@ def __init__(self, handle):
         self._fetch_result = None
 
     def cursor(self):
-        self._cursor = self.handle.cursor()
+        self._cursor = self.handle.cursor(experimental_python_types=True)
```

Comment:
What are the main functionalities of dbt-trino affected by this change? Is it only `dbt seed` ?

Reply:
Have you tested if the PR solves #28 ?

Reply:
Not yet, but will do by next week.

Reply:
Source freshness has been succesfully tested